docs/source/en/glossary.mdx

```diff
@@ -268,34 +306,43 @@ other types of positional embeddings, such as sinusoidal position embeddings or
 ### pretrained model
 
 A model that has been pretrained on some data (for instance all of Wikipedia). Pretraining methods involve a
-self-supervised objective, which can be reading the text and trying to predict the next word (see CLM) or masking some
-words and trying to predict them (see MLM).
+self-supervised objective, which can be reading the text and trying to predict the next word (see [causal language
+modeling](#causal-language-modeling)) or masking some words and trying to predict them (see [masked language
+modeling](#masked-language-modeling)).
```

Comment:
This is too text-centric as well. Other modalities have different pretraining objectives and it would be great to mention them here.